R/FilteredData.R

```diff
@@ -552,23 +554,27 @@ FilteredData <- R6::R6Class( # nolint
       tags$div(
         id = id, # not used, can be used to customize CSS behavior
         class = "well",
+        include_js_files(pattern = "togglePanelItem"),
```

Comment:
Note that we are including the `togglePanelItem` in both `ui_filter_panel` and `ui_active` because teal app only uses the active module directly.